askar_tools/tenant_importer.py

```diff
@@ -217,7 +217,8 @@ async def import_tenant(self):
             )
             async with admin_store.transaction() as admin_txn:
                 wallet_id = str(uuid.uuid4())
-                current_time = time.time()
+                utc_time = datetime.now(timezone.utc)
+                current_time = utc_time.replace(tzinfo=None).isoformat(timespec='microseconds') + "Z"
```

Comment:
In Python 3.12 it seems `datetime.utcnow()` is deprecated?
Think this is the way to get the format like ACA-Py does, but my Python now/utc/timezone/formatting is rusty.

Reply:
The acapy base record the the wallet_record uses generates the timestamp with https://github.com/openwallet-foundation/acapy/blob/main/acapy_agent/messaging/util.py#L14.

I might try and see if this works instead to ensure they are exactly the same.

Reply:
Yeah I saw string format stuff like that `.replace("+00:00", "Z")` as an option searching around too, or stuff like `.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + "Z"` etc

Reply:
Changed to match the syntax from ACA-Py (tested and I get the same date string format)